sandbox/src/mocks/r4/searchServiceRequest/responses/ResponseExampleAdvice.json

```diff
@@ -0,0 +1,71 @@
+{
+    "id": "123e4567-e89b-12d3-a456-426614174002",
+    "resourceType": "Bundle",
+    "meta": {
+        "profile": [
+            "https://fhir.hl7.org.uk/StructureDefinition/UKCore-Bundle"
```

Comment:
We are considering not returning a profile at all in R4. I think @nhsd-jack-wainwright is going to speak to you about this.